roles/tas_single_node/tasks/podman.yml

```diff
@@ -12,10 +12,17 @@
     - "{{ tas_single_node_kube_manifest_dir }}"
     - "{{ tas_single_node_kube_configmap_dir }}"
 
+- name: Podman check login
```

Comment:
Also, please provide a better name for the task - something like "Check whether podman is logged in to the image registry" or similar.